### PR TITLE
Added routing changes on navitems and active style

### PR DESCRIPTION
**Pull request by ianlovell**

### CODE REVIEW

**Comment by csscottlogic** [comment]
:shipit:

File: app/containers/App.jsx [hunk 1]
```diff
@@ -1,28 +1,13 @@
...
-                route: ''
-            }, {
-                title: 'Speakers',
-                route: ''
-            }
-        ]);
-
         return (
             <div>
-                <AppHeader navigation={navigation}/>
```
Comment (jebbinSCL) [inline app/containers/App.jsx]: Where does navigation come from?

Comment (ianlovell) [inline app/containers/App.jsx]: Navigation is now moved into AppHeader, as it's not a reusable component

**Comment by lclarksl** [comment]
The sessions aren't loading on the dashboard for me. Session details view is still working ok though

**Comment by lclarksl** [comment]
Actually I think it's because the sessions service is returning an empty object

**Pull request merged**